contracts/scripts/deployTestRound.ts

```diff
@@ -347,6 +347,95 @@ async function main() {
       { value: deposit }
     )
     recipient3Added.wait()
+
+    // const recipient3Id = await getEventArg(
+    //   recipient3Added,
+    //   recipientRegistry,
+    //   'RequestSubmitted',
+    //   '_recipientId'
+    // )
+    // const executeRequest3 = await recipientRegistry.executeRequest(recipient3Id)
+    // await executeRequest3.wait()
+
+    // const recipient4Added = await recipientRegistry.addRecipient(
+    //   recipient4.getAddress(),
+    //   JSON.stringify(metadataRecipient4),
+    //   { value: deposit }
+    // )
+    // recipient4Added.wait()
+
+    // const recipient4Id = await getEventArg(
+    //   recipient4Added,
+    //   recipientRegistry,
+    //   'RequestSubmitted',
+    //   '_recipientId'
+    // )
+    // const executeRequest4 = await recipientRegistry.executeRequest(recipient4Id)
+    // await executeRequest4.wait()
+
+    // const recipient5Added = await recipientRegistry.addRecipient(
+    //   recipient5.getAddress(),
+    //   JSON.stringify(metadataRecipient5),
+    //   { value: deposit }
+    // )
+    // recipient5Added.wait()
+
+    // const recipient5Id = await getEventArg(
+    //   recipient5Added,
+    //   recipientRegistry,
+    //   'RequestSubmitted',
+    //   '_recipientId'
+    // )
+    // const executeRequest5 = await recipientRegistry.executeRequest(recipient5Id)
+    // await executeRequest5.wait()
+
+    // const recipient6Added = await recipientRegistry.addRecipient(
+    //   recipient6.getAddress(),
+    //   JSON.stringify(metadataRecipient6),
+    //   { value: deposit }
+    // )
+    // recipient6Added.wait()
+
+    // const recipient6Id = await getEventArg(
+    //   recipient6Added,
+    //   recipientRegistry,
+    //   'RequestSubmitted',
+    //   '_recipientId'
+    // )
+    // const executeRequest6 = await recipientRegistry.executeRequest(recipient6Id)
+    // await executeRequest6.wait()
+
+    // const recipient7Added = await recipientRegistry.addRecipient(
+    //   recipient7.getAddress(),
+    //   JSON.stringify(metadataRecipient7),
+    //   { value: deposit }
+    // )
+    // recipient7Added.wait()
+
+    // const recipient7Id = await getEventArg(
+    //   recipient7Added,
+    //   recipientRegistry,
+    //   'RequestSubmitted',
+    //   '_recipientId'
+    // )
+    // const executeRequest7 = await recipientRegistry.executeRequest(recipient7Id)
+    // await executeRequest7.wait()
+
+    // const recipient8Added = await recipientRegistry.addRecipient(
+    //   recipient8.getAddress(),
+    //   JSON.stringify(metadataRecipient8),
+    //   { value: deposit }
+    // )
+    // recipient8Added.wait()
+
+    // const recipient8Id = await getEventArg(
+    //   recipient8Added,
+    //   recipientRegistry,
+    //   'RequestSubmitted',
+    //   '_recipientId'
+    // )
+    // const executeRequest8 = await recipientRegistry.executeRequest(recipient8Id)
+    // await executeRequest8.wait()
```

Comment:
Since we merged some different params into develop for maci params im just going to remove this. To test, you can just go into the vue store and play around with the currentRound